src/main/java/botbot/Botbot.java

```diff
@@ -0,0 +1,47 @@
+package botbot;
+
+import botbot.commands.Command;
+import botbot.commands.ExitCommand;
+import botbot.exceptions.BotbotException;
+
+/**
+ * Represents the chatbot.
+ */
+public class Botbot {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
```

Comment:
Should this class variable be named "uI" instead of "ui"?

> "Variable names must be in camelCase." -[ Java coding standard @ SE-EDU/guides](https://se-education.org/guides/conventions/java/intermediate.html#guides-for-se-student-projects)

I noticed that there could be issues regarding the naming of UI objects in other classes too.